python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -145,7 +145,7 @@ def execute_action(self, data):
                 name=data["name"],
                 entities=entities,
                 project=project_entity,
-                sys_path=sys.path,
+                sys_path=self._compute_sys_path(),
```

Comment:
a small comment here explaining why we cannot just grab it i think could be useful.